nasa-gibs/worldview
# WV-2198: Implement updated design elements to PWA

## Description

This PR adds the new WorldView icons for the PWA and sets the app's theme color to white.

## How To Test

1. After pulling down the changes, run `npm run build && npm start`. 
2. Open [localhost](http://localhost:3000) in chrome and click the "install" icon in the URL.  Looks like an arrow pointing down into a box.
3. You should be prompted to install the app.  It should show the new Worldview globe icon with the white lines running through it.
4. The app should open with a white border around it.

To test on mobile:
1. After running `npm start`, go to your computer's IP address at `:3000` in your phone's browser.  Example:  `192.168.0.123:3000`.
2. Depending on your browser, you may eventually be prompted to download as a PWA.  If not, you can install by going to your browser's "More Options" menu (usually ellipses) and "Add to Home Screen" or similar.
3. Open the app after it has been added to your home screen and you should see a white splash screen with the white Worldview logo in the center.
